scylla/src/transport/retry_policy.rs

```diff
@@ -3,12 +3,12 @@
 //! the `RetryPolicy` trait
 
 use crate::frame::types::Consistency;
-use crate::transport::errors::{DbError, QueryError, WriteType};
+use crate::transport::errors::{DbError, UserRequestError, WriteType};
 
-/// Information about a failed query
-pub struct QueryInfo<'a> {
+/// Information about a failed request
+pub struct RequestInfo<'a> {
```

Comment:
:pick: I'd appreciate this name change in a separate commit, because now it interferes with (rather small) logic changes due to `QueryError` -> `UserRequestError` change.

Reply:
(no need to extract the commit now, it's more of a note for the future)